src/app/app-routing.module.ts

```diff
@@ -6,13 +6,15 @@ import { AddContactComponent } from './pages/add-contact/addContact.component';
 import { ProfileComponent } from './pages/profile/profile.component';
 import { LoginComponent } from './pages/login/login.component';
 import { RegisterComponent } from './pages/register/register.component';
+import { EditProfileComponent } from './pages/edit-profile/editProfile.component';
 import { AuthGuard } from './guards/auth.guard.service';
 
 const routes: Routes = [
   { path: '', canActivate: [AuthGuard], component: HomeComponent },
   { path: 'contact', canActivate: [AuthGuard], component: ContactComponent },
   { path: 'add-contact', canActivate: [AuthGuard], component: AddContactComponent },
   { path: 'profile', canActivate: [AuthGuard], component: ProfileComponent },
+  { path: 'users', canActivate: [AuthGuard], component: EditProfileComponent},
```

Comment:
So since you are doing the profile viewing on one page component and the editing in another component. It's probably best to still to `/profile/edit` as the url path to use.